Tests/BasicsTests/TripleTests.swift

```diff
@@ -9,204 +9,244 @@
-        XCTAssertTriple("i686-apple-macos10.12.0", forPlatformVersion: "13.0", is: "i686-apple-macos13.0")
-
-        XCTAssertTriple("riscv64-apple-darwin", forPlatformVersion: "", is: "riscv64-apple-darwin")
-        XCTAssertTriple("riscv64-apple-darwin", forPlatformVersion: "22", is: "riscv64-apple-darwin22")
-
-        XCTAssertTriple("mips-apple-darwin19", forPlatformVersion: "", is: "mips-apple-darwin")
-        XCTAssertTriple("mips-apple-darwin19", forPlatformVersion: "22", is: "mips-apple-darwin22")
+    @Test
+    func description() throws {
+        let triple = try Triple("x86_64-pc-linux-gnu")
+        #expect("foo \(triple) bar" == "foo x86_64-pc-linux-gnu bar")
+    }
 
-        XCTAssertTriple("arm64-apple-ios-simulator", forPlatformVersion: "", is: "arm64-apple-ios-simulator")
-        XCTAssertTriple("arm64-apple-ios-simulator", forPlatformVersion: "13.0", is: "arm64-apple-ios13.0-simulator")
+    @Test(
+        "Triple String for Platform Version",
+        arguments: [
+            (tripleName: "x86_64-apple-macosx", version: "", expectedTriple: "x86_64-apple-macosx"),
+            (tripleName: "x86_64-apple-macosx", version: "13.0", expectedTriple: "x86_64-apple-macosx13.0"),
+            (tripleName: "armv7em-apple-macosx10.12", version: "", expectedTriple: "armv7em-apple-macosx"),
+            (tripleName: "armv7em-apple-macosx10.12", version: "13.0", expectedTriple: "armv7em-apple-macosx13.0"),
+            (tripleName: "powerpc-apple-macos", version: "", expectedTriple: "powerpc-apple-macos"),
+            (tripleName: "powerpc-apple-macos", version: "13.0", expectedTriple: "powerpc-apple-macos13.0"),
+            (tripleName: "i686-apple-macos10.12.0", version: "", expectedTriple: "i686-apple-macos"),
+            (tripleName: "i686-apple-macos10.12.0", version: "13.0", expectedTriple: "i686-apple-macos13.0"),
+            (tripleName: "riscv64-apple-darwin", version: "", expectedTriple: "riscv64-apple-darwin"),
+            (tripleName: "riscv64-apple-darwin", version: "22", expectedTriple: "riscv64-apple-darwin22"),
+            (tripleName: "mips-apple-darwin19", version: "", expectedTriple: "mips-apple-darwin"),
+            (tripleName: "mips-apple-darwin19", version: "22", expectedTriple: "mips-apple-darwin22"),
+            (tripleName: "arm64-apple-ios-simulator", version: "", expectedTriple: "arm64-apple-ios-simulator"),
+            (tripleName: "arm64-apple-ios-simulator", version: "13.0", expectedTriple: "arm64-apple-ios13.0-simulator"),
+            (tripleName: "arm64-apple-ios12-simulator", version: "", expectedTriple: "arm64-apple-ios-simulator"),
+            (tripleName: "arm64-apple-ios12-simulator", version: "13.0", expectedTriple: "arm64-apple-ios13.0-simulator")
+        ]
+    )
+    func tripleStringForPlatformVersion(tripleName: String, version: String, expectedTriple: String) throws {
+           guard let triple = try? Triple(tripleName) else {
```

Comment:
```suggestion
           let triple = try #require(try Triple(tripleName))
```

Reply:
It was a PEBCAK issue.. fixed locally.

Reply:
This compiler error is happening because `isAppleIsDarwin` isn't marked as `throws`. If you annotate the function with `throws` the error will propagate properly.